src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -409,15 +409,25 @@ ValidatorList::for_each_listed (
 }
 
 std::size_t
-ValidatorList::calculateQuorum (std::size_t nTrustedKeys)
+ValidatorList::calculateMinimumQuorum (
+    std::size_t nListedKeys, bool unlistedLocal)
 {
-    // Use 80% for large values of n, but have special cases for small numbers.
-    constexpr std::array<std::size_t, 10> quorum{{ 0, 1, 2, 2, 3, 3, 4, 5, 6, 7 }};
-
-    if (nTrustedKeys < quorum.size())
-        return quorum[nTrustedKeys];
-
-    return nTrustedKeys - nTrustedKeys / 5;
+    if (nListedKeys == 0)
```

Comment:
I think @wilsonianb noted that standalone mode operates with an empty list.  In that case, it would never fully validate a ledger.   I'm not sure if that is a problem or not.  If anything, that might be better configured via the "EXPERT ONLY" minimumQuorum, and leave this returning 1?